account-compression/programs/account-compression/src/macros.rs

```diff
@@ -0,0 +1,106 @@
+/// This macro applies functions on a ConcurrentMerkleT:ee and emits leaf information
+/// needed to sync the merkle tree state with off-chain indexers.
+#[macro_export]
+macro_rules! merkle_tree_depth_size_apply_fn {
```

Comment:
nit: how about making this private too, ie 
```suggestion
macro_rules! _merkle_tree_depth_size_apply_fn {
```

Reply:
Updated!